src/content/resources/faq.md

```diff
@@ -357,6 +356,16 @@ developers to access new mobile OS features and capabilities
 immediately. Developers don't have to wait for the Flutter team
 to expose the new mobile OS capability.
 
+### Does Flutter support code push?
+
+Code push, or the ability to push app updates directly to a user's device,
+isn't directly supported by Flutter. However, there is a third
```

Comment:
potentially we could say something like "However, we're aware of
at least one third-party solution for code push for Flutter called Shorebird. Note that this is not
an official endorsement or recommendation."